addons/l10n_za/data/template/account.account-za.csv

```diff
@@ -100,9 +104,10 @@
 "610410","610410","Insurance","expense","False"
 "610420","610420","Travel and subsistence","expense","False"
 "610430","610430","Corporation tax expense","expense","False"
-"610440","610440","Foreign Exchange Gains/Losses","expense","False"
```

Comment:
This line is not a rename, the " 610440 Foreign Exchange Gains/Losses" was replaced by 424020 for the gains and 610430 for the Losses. This line is actually a new one that takes the account number 610440 which is now free.